### ISSUE TITLE
Dependabot alert: `libsqlite3-sys` via C SQLite improperly validates array index

### ISSUE THREAD

**Issue by josecelano** [torrust/torrust-index#124]
GitHub Advisory Database: https://github.com/advisories/GHSA-jw36-hf63-69r9

```
$ cargo tree -i -p libsqlite3-sys
libsqlite3-sys v0.24.2
├── sqlx-core v0.6.3
│   └── sqlx-macros v0.6.3 (proc-macro)
│       └── sqlx v0.6.3
│           └── torrust-index-backend v2.0.0-dev.1 (/home/josecelano/Documents/git/committer/me/github/torrust/torrust-index-backend)
└── sqlx-core v0.6.3
    └── sqlx v0.6.3 (*)
```

[sqlx](https://github.com/launchbadge/sqlx) crate is using it.

It seems it has been fixed but not included in a new release yet https://github.com/launchbadge/sqlx/issues/2174

**Comment by da2ce7** [comment]
Should have been fixed in the 0.7 release: https://github.com/launchbadge/sqlx/releases/tag/v0.7.0

**Comment by josecelano** [comment]
Fixed via https://github.com/torrust/torrust-index/compare/272c384f06...c5f11a11b2
